templates/OperationResponseFailureSnippet.stencil

```diff
@@ -0,0 +1,5 @@
+{% if op.defaultExceptionHasBody == true %}
```

Comment:
Maybe this file could be called "OperationDefaultExceptionSnippet" since that's what it is doing.

Reply:
Thanks. Renamed.